sqlstore/src/main/resources/db/migration/V0163__auto_site.sql

```diff
@@ -1,13 +1,7 @@
--- harmonize_TGL_kd_ts
 -- StartNoTest
---StartNoTest
-CALL addTargetedSequencing('Agilent SureSelectXT MethylSeq', 'TGL', 'Agilent SureSelectXT MethylSeq', 0);
-CALL addTargetedSequencing('IDT xGEN Exome Research Panel v1', 'TGL', 'IDT xGEN Exome Research Panel v1', 0);
-CALL addTargetedSequencing('TruSeq RNA Access', 'TGL', 'TruSeq RNA Access', 0);
-CALL addTargetedSequencing('Roche SeqCap Epi CpGiant', 'TGL', 'Roche SeqCap Epi CpGiant', 0);
-CALL addTargetedSequencing('TruSeq Methylation Epic Exome', 'TGL', 'TruSeq Methylation Epic Exome', 0);
-
-CALL addKitDescriptor('Agilent SureSelect Human All Exon V5 + UTRs', 1, 'Agilent', 1, 'Library', 'Illumina', 'TGL');
+INSERT INTO KitDescriptor(name, version, manufacturer, partNumber, kitType, platformType, description, lastModifier)
+  SELECT 'Agilent SureSelect Human All Exon V5 + UTRs', 1, 'Agilent', 1, 'Library', 'Illumina', 'TGL', getAdminUserId() FROM DUAL
```

Comment:
If we can guarantee it will exist, I'm willing to believe the API is stable, no?